src/java.desktop/windows/native/libawt/windows/awt_TrayIcon.cpp

```diff
@@ -70,10 +70,12 @@ typedef struct tagBitmapheader  {
 
 jfieldID AwtTrayIcon::idID;
 jfieldID AwtTrayIcon::actionCommandID;
+jmethodID AwtTrayIcon::updateImageDpiID;
```

Comment:
I'd rather drop 'Dpi' from the name because the name of the Java method is `updateImage`.

If you prefer, the C++ class method could also be changed to `UpdateImage`, dropping 'DPI'.

Reply:
yes, that's a good point on the naming